projects/dsp-ui/src/lib/search/expert-search/expert-search.component.scss

```diff
@@ -0,0 +1,32 @@
+.expert-search-container {
+  //   min-height: 100%;
+
+  .expert-search-form {
+    min-width: 150px;
+    width: 100%;
+    margin: auto;
+
+    .textarea-field {
+      width: 100%;
+      display: block;
+      margin-bottom: 24px;
+    }
+
+    .form-panel {
+      width: 100%;
+    }
+  }
+}
+
+.mat-input-element {
+  font-family: "Courier New", Courier, monospace;
+}
+
+.form-content {
+  margin: 24px auto;
+  width: 472px;
+
+  .large-field {
+    min-width: 472px;
+  }
+}
```

Comment:
Can you add a TODO comment here? The general form settings should degined in a main stylesheet in the module's root folder. But this should be part of a separate PR.

Reply:
done in 59c1d2d